BUILDING.md

```diff
@@ -23,6 +23,37 @@ For Linux the instructions for Ubuntu are provided, but you can find the equival
 sudo apt-get install cmake ninja-build libsdl2-dev libdbus-1-dev libfreetype-dev lld llvm clang
 ```
 
+### macOS
+Building on macOS requires Homebrew (a package manager for macOS), a special MIPS-capable version of Clang, and Xcode for Metal shader support. Follow these steps:
+
+1. **Install Homebrew** (if not already installed):
+   - Open Terminal and run:
+     ```bash
+     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+     ```
+   - Follow the on-screen instructions to complete the installation.
+

```

Comment:
Would prefer if this just linked to the Homebrew site instead of embedding the install instructions, that way it doesn't need to be kept up to date here.